test/memspaces/memspace_helpers.hpp

```diff
@@ -16,6 +16,8 @@
 #define SIZE_4K (4096UL)
 #define SIZE_4M (SIZE_4K * 1024UL)
 
+#define IS_SKIPPED_OR_FAILED() (HasFatalFailure() || IsSkipped())
```

Comment:
hmm you can change this later (with other PR or something) - but is this helper function only needed in `memspace_helpers.hpp`? no other tests will use it...?

Reply:
that was just a thought... perhaps `base.hpp` if you wantto put it in hpp header